src/Money.php

```diff
@@ -539,4 +482,28 @@ public function jsonSerialize()
             'currency' => $this->currency,
         );
     }
+
+    /**
+     * @param Calculator[] $calculators
+     */
+    public static function registerCalculators(array $calculators)
```

Comment:
I think having a singular method would be better. I would also add a check whether the class exists at all and implements our Calculator interface. Also, why default? It could be just calculators IMO